Platforms/QemuQ35Pkg/PlatformBuild.py

```diff
@@ -25,14 +25,14 @@
 
 # Declare test whose failure will not return a non-zero exit code
 failure_exempt_tests = {}
-failure_exempt_tests["BaseCryptLibUnitTestApp.efi"] = datetime.datetime(2023, 3, 7, 0, 0, 0)

```

Comment:
Is this being removed because it fails? If so, is there a line of sight to getting it fixed?

Reply:
Ken fixed it a few weeks ago

Reply:
Oh, I see, my misunderstanding. Not each test goes here, just ones that are currently allowed to fail.